cicd/ReleasePipeline.jenkinsfile

```
@@ -0,0 +1,75 @@
+def appName = 'casino'
+def repositoryName = 'casino'
+
+// TODO: Change target channel for branch deployment notifications
+//       Make sure target channel has Jenkins app added to it
+def slackChannel = '#cicd-production-deployments'
+def buildLink = "<${env.RUN_DISPLAY_URL}|${env.JOB_NAME} ${env.BUILD_DISPLAY_NAME}>"
+def slackResponse = slackSend(channel: slackChannel, message: "*${repositoryName} | ${buildLink}*\nPreparing full release...", color: "#0099ff")
+
+pipeline {
+  agent { label 'dind' }
+
+  environment {
+    GITHUB_API_TOKEN = credentials('github-svc-cicd')
+    JIRA_API_TOKEN = credentials('jira-svc-cicd')
+    DATADOG_API_TOKEN = credentials('datadog-api-token')
+    DATADOG_APPLICATION_KEY = credentials('datadog-application-token')
+    CHANGELOG_VERSION = sh(script: "grep -oE -m 1 '([0-9]+)\\.([0-9]+)\\.([0-9]+)' CHANGELOG.md", returnStdout: true).trim()
+    RELEASE_VERSION = "${CHANGELOG_VERSION}-${BUILD_NUMBER}"
+    COMMIT_AUTHOR = sh(script: "git log --format='%an' ${env.GIT_COMMIT}^!", returnStdout: true).trim()
+    COMMIT_AUTHOR_EMAIL = sh(script: "git log --format='%ae' ${env.GIT_COMMIT}^!", returnStdout: true).trim()
+    COMMIT_LINK = "<${env.GIT_URL.replace('.git', '')}/commit/${env.GIT_COMMIT}|${env.GIT_COMMIT.substring(0, 7)}>"
+    BUILD_TITLE = "*<${env.GIT_URL}|${repositoryName}> :git: | ${buildLink}*"
+  }
+
+  // Configuration that applies to the whole job
+  options {
+    ansiColor('xterm') // Beautiful console colors
+    timeout(time: 3, unit: 'HOURS') // Set a build timeout
+  }
+
+  stages {
+    stage('TagRelease')  {
+      agent { label 'release-helper' }
```

Move to root level because `dind` isn't used
```suggestion
      agent { label 'release-helper' }
```

---

Think I have applied this as intended